src/app/shared/constants/validation-messages.ts

```
@@ -0,0 +1,55 @@
+  static readonly INVALID_AGE_RANGE = 'FORMS.VALIDATIONS.INVALID_AGE_RANGE';
+  static readonly INVALID_VALUE = 'FORMS.VALIDATIONS.INVALID_VALUE';
+  static readonly INVALID_SECTION_NAME = 'FORMS.VALIDATIONS.INVALID_SECTION_NAME';
+  static readonly MUST_CONTAIN_LETTERS = 'FORMS.VALIDATIONS.MUST_CONTAIN_LETTERS';
+  static readonly LESS_THAN_PARTICIPANTS = 'FORMS.VALIDATIONS.LESS_THAN_PARTICIPANTS';
+  static readonly INVALID_EMAIL_TYPE = 'FORMS.VALIDATIONS.INVALID_EMAIL_TYPE';
+  static readonly INVALID_LENGTH_FROM_TO = 'FORMS.VALIDATIONS.INVALID_LENGTH_FROM_TO';
+  static readonly INVALID_LENGTH_FROM_TO_WITH_COUNT = 'FORMS.VALIDATIONS.INVALID_LENGTH_FROM_TO_WITH_COUNT';
+  static readonly INVALID_LENGTH_NO_MORE_THAN = 'FORMS.VALIDATIONS.INVALID_LENGTH_NO_MORE_THAN';
+  static readonly INVALID_LENGTH_NO_MORE_THAN_WITH_COUNT = 'FORMS.VALIDATIONS.INVALID_LENGTH_NO_MORE_THAN_WITH_COUNT';
+  static readonly INVALID_TAGS_LENGTH = 'FORMS.VALIDATIONS.INVALID_TAGS_LENGTH';
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)